app/src/main/res/values/strings.xml

```diff
@@ -594,4 +592,10 @@
     <string name="choose_instance_prompt">Choose an instance</string>
     <string name="app_language_title">App language</string>
     <string name="systems_language">System default</string>
+    <string name="dynamic_seek_duration_description">%s seconds</string>
+    <plurals name="dynamic_seek_duration_description">
+        <item quantity="few">%s seconds</item>
+        <item quantity="many">%s seconds</item>
+        <item quantity="other">%s seconds</item>
+    </plurals>
```

Comment:
Can you test whether "other" is enough here?
`res.getQuantityString(R.plurals.dynamic_seek_duration_description, currentDurationValue)` should pick the correct one when "few" or "many" is needed by localization.

Reply:
I already tested with esperanto, and yeah other is enough. But for English I left all three, because it's the source string. I don't know if it's necessary, or if weblate will create «many» or «few» for other languages, do you know?

Reply:
Yes, weblate knows which language has supports the quantities.

Reply:
ok, done